### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by tcuc**
add dockerfile
add docker build instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,5 +28,13 @@ $ npm install
 $ npm run start
 ```
 
+_To build/run docker container:_
+```bash
+$ cd wordle
+$ docker build -t notwordle .
+$ docker run -d -p 3000:3000 notwordle
+```
+open http://localhost:3000 in browser.
```
Comment (hannahcode) [inline README.md:37]: It appears this is using your file directory structure, can we make this generic?

**Review by hannahcode — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tcuc) [reply README.md:37]: I wrote it relative to the repo root dir.
And i made the assumption that it is running locally.
So I'm not quite sure what else could be done more generic?
I also admit that i may be overlooking something obvious, so please let me know if I'm overlooking something. :slightly_smiling_face:

Comment (Bockiii) [reply README.md:37]: when this is merged, I'll open a PR to do github autobuilds and host the container in the ghcr. Then nobody should really need to locally build it.

Comment (tcuc) [reply README.md:37]: Sweet!

Comment (hannahcode) [reply README.md:37]: Got it - lets remove the "cd wordle" line, because we can assume the user is already in the directory?

Comment (Bockiii) [reply README.md:37]: sure

Comment (tcuc) [reply README.md:37]: I agree :)

**Pull request merged**